mazestyle.css

```diff
@@ -43,4 +43,10 @@ canvas {
     margin-right: auto;
     display: block;
     border: 2px solid blue;
+}
+
+@media only screen and (max-width: 757px) {
+    #canvas {
+        width: 85%;
+    }
 }
```

Comment:
Looks good! Can you add a newline at the end of the file? I am getting a "No newline at the end of the file" warning with your commit. Just go to the last line and hit enter!

Reply:
Just made the change, let me know if you're still getting the error!

Reply:
Hey I am still getting the error! From my understanding, if line 52 is the last line of the file there basically needs to be a blank line 53 at the end (essentially a newline)

Reply:
Just gave it another shot, should leave a space at line 53.